### DISCUSSION TITLE
Python sdk in the Event log

### DISCUSSION THREAD

**Discussion by gurosatherHeco** [Volue-Public/energy-mesh-python#327] [Q&A] [answered]
Hi,
We can not see the pyhton sdk activity in the event log anymore?
In september the python sdk activity was shown as Mesh sessions as below in the event log:
![image](https://user-images.githubusercontent.com/82815172/198585569-9cc301bc-3274-44cd-a2d5-5e182695a8d4.png)

**Comment by tnoczyns-volue** [comment, marked as answer]
Hi,

Writing logs to Event Log is done by the Mesh server and should work correctly. For reading time series you should see something similar to:
![image](https://user-images.githubusercontent.com/88330807/198633938-d8b5df63-c339-4a99-b915-ae26f90a1263.png)

For writing to Event Log, Mesh server needs to be configured to use Oracle database. Could you verify that in the Mesh configuration file?
On what version of Mesh and Python SDK are you observing the issue? What has changed since September?

**Reply by alftester** [reply]
We see this on Mesh version 2.6.1+8 and from pyproject.toml I see version 1.0.0.

**Reply by tnoczyns-volue** [reply]
There is a bug in Mesh server, where Event Logger does not recover after loss of Oracle connection. We are working on fixing that. Meanwhile I'd suggest Mesh server restart if the issue is observed.

**Reply by alftester** [reply]
We'll do a restart, to see if this bug may be the issue. Thanks.

**Reply by alftester** [reply]
@tnoczyns-volue  A restart of the server solved the issue. Thanks again.
